### DISCUSSION TITLE
Displaying quantities in SI base units

### DISCUSSION THREAD

**Discussion by b00nw33** [sympy/sympy#24320] [Q&A]
Hi all. I just started using Sympy this week. According to various documentation, including this [video](https://www.youtube.com/watch?v=fwgjeNlcJ4o), when the following code was run in the past, the units of ohm would be returned in SI base units.
```
import sympy.physics.units as u
u.ohm
```
However, when I tried, I got the symbol omega instead. Is there a method to obtain the same SI base units $\frac{kgm^2}{A^2s^3}$? It seems like this form of output was removed, but it would be useful if it can be retrieved as a method.

Versions used:
```
IPython console for SymPy 1.6.2 (Python 3.8.10-64-bit) (ground types: gmpy)
```

Thank you

**Comment by oscarbenjamin** [comment]
There might be an easier way but you can do it like this:
```python
In [70]: import sympy.physics.units as u

In [71]: u.convert_to(u.ohm, [u.amperes, u.kg, u.s, u.m])
Out[71]: 
              2
kilogram⋅meter 
───────────────
      2       3
ampere ⋅second 
```
